docs/user/power/install_upi.md

```diff
@@ -0,0 +1,300 @@
+# Install: Power User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Example automation is provided [here](#example-power-upi-configuration) to help model your own.
+
+## Table of contents
+
+1. Minimum compute requirements
+
+2. Network topology requirements
+
+3. DNS requirements
+
+4. Getting Ignition configs for machines
+
+5. Getting OS related assets for machines
+
+6. Booting machines with RHCOS and Ignition configs
+
+7. Watching your installation (bootstrap complete, cluster available)
+
+8. Example Bare-Metal UPI deployment
+
+## Compute
+
+The smallest OpenShift 4.x clusters require the following host:
+
+* 1 bootstrap machine.
+
+* 3 control plane machines.
+
+* atleast 1 worker machine.
+
+NOTE: The cluster requires the bootstrap machine to deploy the OpenShift cluster on to the 3 control plane machines, and you can remove the bootstrap machine.
+
+All the machines must use Red Hat Enterprise Linux CoreOS (RHCOS) as the operating system.
+
+### Minimum resource requirements
+
+| Machine       | Operating System | vCPU | RAM   | Storage |
+|---------------|------------------|------|-------|---------|
+| Bootstrap     | RHCOS            | 2    | 16 GB | 120 GB  |
+| Control Plane | RHCOS            | 2    | 16 GB | 120 GB  |
+| Compute       | RHCOS            | 2    | 16 GB | 120 GB  |
+
+
+
+## Network Topology Requirements
+
+The easiest way to get started is to ensure all Power nodes have internet access to pull images for platform containers and provide telemetry data to Red Hat.
+OpenShift 4.x also supports air-gapped installs
+
+### Load balancers
+
+Before you install OpenShift, you must provision two load balancers.
+
+* A load balancer for the control plane and bootstrap machines that targets port 6443 (Kubernetes APIServer) and 22623([Machine Config server][machine-config-server]). Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster, and port 22623 must be accessible to nodes within the cluster.
+
+  NOTE: Bootstrap machine can be deleted as target after cluster installation is finished.
```

Comment:
What does "deleted as target mean"?

Reply:
my bad.. should have changed it to "Bootstrap machine can be deleted after cluster installation is finished."